checkov/terraform/tf_parser.py

```diff
@@ -461,6 +462,7 @@ def parse_hcl_module_from_tf_definitions(
         )
         self.add_tfvars(module, source)
         copy_of_tf_definitions = pickle_deepcopy(tf_definitions)
+        module.temp_tf_definition = tf_definitions  # type:ignore  # will be TFDefinitionKey and not string
```

Comment:
why ignore instead of changing the type in line 51?

Reply:
Already tried to play with it a couple of times and it didn't work, (u can see at the first couple of commits).